addons/website/views/website_templates.xml

```diff
@@ -1828,6 +1828,11 @@
 </template>
 
 <template id="language_selector" inherit_id="portal.language_selector">
+    <!-- Language selectors don't need to be folded in the auto-hide menu -->
+    <xpath expr="//*[contains(@t-attf-class, 'js_language_selector')]" position="attributes">
+        <attribute name="t-attf-class" add="o_no_autohide_item" separator=" "/>
+    </xpath>
```

Comment:
Can't you target the `js_language_selector` class in your JS? That would avoid this xpath and XML change